presto-atop/src/main/java/io/prestosql/plugin/atop/AtopTable.java

```diff
@@ -70,15 +71,15 @@
 
     private final String name;
     private final String atopLabel;
-    private final List<AtopColumn> columns;
-    private final Map<String, AtopColumn> columnIndex;
+    private final ImmutableList<AtopColumn> columns;
```

Comment:
We always interface types like `List`, not `ImmutableList`. Some code bases do use the immutable classes as types, but we do not, as we consider that an implementation detail.

Reply:
error-prone force us to use `ImmutableList`, or it will violate error-prone immutable rule, even if the collection is immutable.

Reply:
Error prone has some interesting opinions, but that simply isn't how we write code in this project.  Does error prone have configuration were we can disable this behavior?